packages/manager/modules/pci/src/projects/project/additional-ips/order/order.controller.js

```diff
@@ -241,9 +242,9 @@ export default class AdditionalIpController {
   initIp() {
     const plan = this.getFailoverIpPlan();
     const floatingIpPrice = `${plan.pricings[0].price + plan.pricings[0].tax}${
-      this.currencySymbol
+      this.user.currency.symbol
     }`;
-    const failoverIpPrice = `0${this.currencySymbol}`;
+    const failoverIpPrice = `0${this.user.currency.symbol}`;
```

Comment:
The price is always 0 ?
Isn't it the api that gives the price?

Reply:
I am not sure regarding this point. But this change appeared because of refactor. Is it okay if I can check this and create a new ticket if required and ignore this point as part of current PR?